connectivity/FEATURE_BLE/source/generic/GapImpl.cpp

```diff
@@ -3042,6 +3058,15 @@ void Gap::on_private_address_generated(bool connectable)
         return;
     }
 
+    if (_privacy_initialization_pending &&
+        _address_registry.get_resolvable_private_address() != address_t{} &&
```

Comment:
is that possible?

Reply:
Yes it is if privacy is enabled then disabled or if the security manager has been enabled ~50ms before the privacy.

Reply:
how would the on_private_address_generated be called then?

Reply:
It is called but it has no effect since privacy is not enabled.